src/js/node/Fs.hx

```diff
@@ -95,6 +95,14 @@ typedef FsWriteFileOptions = {
 	  autoClose: true
 	}
 **/
+/**
+ * options for `Fs.mkdir` and `Fs.mkdirSync`
+ */
+typedef FsMakeDirOptions = {
```

Comment:
I think this should be called `FsMkdirOptions` to be consistent with the function names :)